WebService/Runtime/Config.cs

```diff
@@ -170,6 +170,30 @@ private static IServicesConfig GetServicesConfig(IConfigData configData)
                 // file, or in the PCS_IOTHUB_CONNSTRING environment variable. When
                 // working with VisualStudio, the environment variable can be set in the
                 // WebService project settings, under the "Debug" tab.
+                Console.BackgroundColor = ConsoleColor.Red;
+                Console.ForegroundColor = ConsoleColor.Yellow;
```

Comment:
nit: consider moving code that prints output to console (lines 173 - 195) to a private helper method.